src/main/java/com/data2viz/kotlinx/htmlplugin/ide/controller/ConvertHTMLToKotlinXAction.java

```diff
@@ -0,0 +1,9 @@
+package com.data2viz.kotlinx.htmlplugin.ide.controller;
```

Comment:
Package must be the same as the company domain: `io.data2viz`

Reply:
ok